lib/features/children/overview/pages/children_overview_page.dart

```diff
@@ -35,7 +36,13 @@ class ChildrenOverviewPage extends StatelessWidget {
           children: [
             Row(
               children: [
-                BackButton(),
+                BackButton(
+                  onPressed: () {
+                    Navigator.maybePop(context);
```

Comment:
Why are you using the Navigator class? As we use the go_router for this? Like "context.pop()"